xbmc/utils/FileExtensionProvider.h

```diff
@@ -77,15 +78,25 @@ class CFileExtensionProvider
   void SetAddonExtensions(ADDON::AddonType type);
 
   void OnAddonEvent(const ADDON::AddonEvent& event);
+  void OnAdvancedSettingsLoaded();
 
   // Construction properties
   std::shared_ptr<CAdvancedSettings> m_advancedSettings;
   ADDON::CAddonMgr &m_addonManager;
+  int m_callbackId;
```

Comment:
default init the member

Reply:
It's initialized in the constructor with the other members.